src/distributions/uniform.rs

```diff
@@ -173,6 +174,9 @@ pub trait UniformImpl: Sized {
 }
 
 /// Implementation of `UniformImpl` for integer types.
+///
+/// Unless you are implementing `RangeImpl` for your own type, this type should
+/// not be used directly, use `Range` instead.
```

Comment:
You forgot to update the `Range` names

Reply:
Sorry, this is a left-over from the merge...